src/components/IncidentsTable.tsx

```diff
@@ -4,6 +4,8 @@ import { useNavigate } from 'solid-app-router'
 import { Incident } from '../types/incident'
 import Button from './Button'
 import IncidentRow from './IncidentRow'
+import { Portal } from 'solid-js/web'
+import CreateIncidentForm from './CreateIncidentForm'
 
 type Props = {
   incidents: Accessor<{ incidents: Incident[] }>
```

Comment:
We should be able to clean this `Accessor` up with the new pattern we came up with.